test/functional/fixtures/runner/using-shared-code/shared-code.test.js

```diff
@@ -0,0 +1,48 @@
+'@fixture Using shared code in the iframe pages';
+'@page ./base-page.html';
+
+'@require :testModule';
+'@require ./mixin.js';
+
+
+var basePath = 'http://localhost:3000/runner/using-shared-code/';
+var text     = 'test';
+
+
+'@test'['Using shared code in iframe'] = {
+    '@testCases': [
+        { '@name': 'Cross-domain iframe', pageUrl: 'cross-domain.html' },
+        { '@name': 'Same-domain iframe', pageUrl: 'same-domain.html' },
+        { '@name': 'Inline iframe', pageUrl: 'inline-iframe.html' }
+    ],
+
+    '1.Navigate to the test page': function () {
+        act.navigateTo(basePath + this.pageUrl);
+    },
+
+    '2.Type in input': inIFrame('#iframe', function () {
+        act.type(getInput, text);
+    }),
+
+    '3.Check input value': inIFrame('#iframe', function () {
+        eq($('#input')[0].value, 'test');
+    })
+};
+
+'@test'['Using mixin in iframe'] = {
+    '@testCases': [
+        { '@name': 'Cross-domain iframe', pageUrl: 'cross-domain.html' },
+        { '@name': 'Same-domain iframe', pageUrl: 'same-domain.html' },
+        { '@name': 'Inline iframe', pageUrl: 'inline-iframe.html' }
+    ],
+
+    '1.Navigate to the test page': function () {
```

Comment:
Let's omit "the" as we omit articles in other step names. So, it should be:
_Navigate to test page_